### PR TITLE
drop Makefile, update README

### LINKED ISSUE #5478: README documentation out of date
### Steps to reproduce

1. Read the README.md
2. Clone the repo
3. Run the command `make`

### Expected behavior

As a new contributor to Calendar, I want to easily following instructions in the README.md file of how to start developing. 

Currently, I am stuck on the first step of `Build the app` documentation, as the `make` command does not complete successfully.

### Actual behaviour

After many steps `make` seems to complete correctly, but the `make` command ends with an error about about a missing `phpunit.xml` file.

### Calendar app version

3be608530450eb509b38d761b4003e369413ecf7

### CalDAV-clients used

_No response_

### Browser

_No response_

### Client operating system

_No response_

### Server operating system

Debian 12

### Nextcloud version

28

### Updated from an older installed version or fresh install

fresh install

### Web server error log

```Text
...

Test Suites: 60 passed, 60 total
Tests:       356 passed, 356 total
Snapshots:   25 passed, 25 total
Time:        63.094 s
Ran all test suites.
phpunit -c phpunit.xml
Could not read "phpunit.xml".
make: *** [Makefile:43: test-php] Error 1
```
```


### Log file

_No response_

### Browser log

_No response_

### Additional info

Console error log

```
...

Test Suites: 60 passed, 60 total
Tests:       356 passed, 356 total
Snapshots:   25 passed, 25 total
Time:        63.094 s
Ran all test suites.
phpunit -c phpunit.xml
Could not read "phpunit.xml".
make: *** [Makefile:43: test-php] Error 1
```

**Observations and debugging**

- Using the 2 year old `phpunit.unit.xml` fails with errors
- Running `composer run test` seems to complete successfully

### PR DESCRIPTION
**Pull request by privatemaker**
With gracious guidance from @ChristophWurst these changes "worked for me" and seem to be current across apps 😄

 fixes #5478

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Awesome

File: README.md [hunk 1]
```diff
@@ -47,30 +47,28 @@ If you find bugs or have problems with the CalDAV-Backend, you should ask the te
 
 ## Build the app
 
-Make sure you have all the [necessary prerequisites](https://docs.nextcloud.com/server/latest/admin_manual/installation/source_installation.html#prerequisites-for-manual-installation) for manual installs
+General build instructions for the app itself are the same for all options.
```
Comment (tcitworld) [inline README.md:50]: **question:** What does `all options` mean here?

**Review by tcitworld — CHANGES_REQUESTED** [review]
Most other NC projects still have Makefiles as well, I would wait for different opinions (especially from the ones who made the work at https://nextcloud.com/contribute/ and https://nextcloud.com/developer/ first to see what they prefer).

Otherwise we can just fix it if we keep it

**Review by miaulalala — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (privatemaker) [reply README.md:50]: I don't understand your suggestion nor what `all option` means here. That is not what is in the `README.md` file in my pull-request.

**Comment by privatemaker** [comment]
> Most other NC projects still have Makefiles as well

@tciworld the `Makefile` is broken and relying on files that no longer exist. It is my understanding from @ChristophWurst and other devs on server team the current approach of NC is to move away from `Makefiles` entirely and instruct users only to use lang specific tools manually (`composer` and `npm`).

**Comment by ChristophWurst** [comment]
We chatted about it here at the conference and the developer's view seems to be that we want people to know npm and composer because they need these tools to do any meaningful changes. The makefile adds an unnecessary abstraction.

Ref https://github.com/nextcloud/documentation/pull/11121

**Review by tcitworld — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (privatemaker) [reply README.md]: @tcitworld I was looking at the wrong part of the README due to github coloring. I had copied the block from Deck app and the `all options` line was an oversight.

Apologies for confusion, thanks for catching my mistake 😄

**Review by st3iny — APPROVED** [review]

**Review by miaulalala — APPROVED** [review]

**Pull request merged**